machines/vcItem.ts

```diff
@@ -1003,16 +1007,26 @@ export const vcItemMachine =
           { to: (context) => context.serviceRefs.store }
         ),
 
-        loadMyVcs: send(StoreEvents.GET(MY_VCS_STORE_KEY), {
-          to: (context) => context.serviceRefs.store,
-        }),
-
         removeVcItem: send(
           (_context, event) => {
             return StoreEvents.REMOVE(MY_VCS_STORE_KEY, _context.vcKey);
           },
           { to: (context) => context.serviceRefs.store }
         ),
+
+        storingActivityLog: send(
```

Comment:
for now: can this function be named `logVCremoved` as then this would be in-line with other names such as `logWalletBindingSuccess` `logWalletBindingFailure`, etc.

for much later(maybe as a separate Jira ticket in a future sprint): this and other similar functions can be thought to be refactored into a utility function where-in only the event-type and context object references are sent

Reply:
@srikanth716 Is this for capturing all activity logs or just VC removed?
If it's VC removed, then we should have function name as harsh is suggesting.

Reply:
This only for VC removed we can change the name as harsh is suggesting